server/src/main/java/com/distkv/dst/server/runtime/workerpool/Worker.java

```diff
@@ -78,9 +112,31 @@ public void run() {
           }
           case STR_DROP: {
             CommonProtocol.DropRequest request =
-                    (CommonProtocol.DropRequest) internalRequest.getRequest();
+                (CommonProtocol.DropRequest) internalRequest.getRequest();
+            if (isMaster) {
+              for (SalverClient client : salverClients) {
```

Comment:
I think it's better to add a helper method like `syncToSlaves()` to avoid duplicated code:

```java
Status syncReuqestToSlaves<RequestType, ResponseType>(clients, RequestType request, lambda operation) {
}
```